doc/ChangeLog.md

```diff
@@ -30,6 +30,9 @@ All notable changes to the project are documented in this file.
  - Fix #956: CLI `copy` command complains it cannot change owner when
    copying `factory-config` to `running-config`.  Bogus error, the
    latter is not really a file
+ - Fix #936: DHCP server reconfiguration does not always take effect.
+ - Fix #917: ietf-keystore: Unsupported key types are not deviated in
+   the YANG model.
 
```

Comment:
First, and it's only a minor thing, we used to sort all fixed-issues (and changes with issue #) numerically.

Second, the fix to #917, I believe is listed under Changes as "YANG type change", but best check with @mattiaswal about that detail.

Other than that, looks good, so I'm approving this and leaving it up to your discretion to re-sort issues and/or adjust for the ietf-keystore comment.

Reply:
I thing that line is wrong, it should be more like "ietf-keystore: YANG Type has changed for SSH private and public keys"

Reply:
It was already there, remove the line with #917, it does not give the reader anything at all.

Reply:
I took the original line directly from the issue #917 . I just updated it to your quote @mattiaswal . Happy? :-)